tests/test.js

```diff
@@ -0,0 +1,20 @@
+const ledger = require('./ledger-node')
+const Zemu = require('@zondax/zemu').default
+
+// Error: elfPath cannot be null!
+// Need a default app to use here...
+const sim = new Zemu();
```

Comment:
@jleni Is there a way to start Zemu without an app ?

Reply:
not at the moment. 
We could create an empty app in Zemu, that replies to "getVersion" and we can use that for testing.

Reply:
I noticed there is a demoApp (https://github.com/Zondax/zemu/tree/master/bin/demoApp) and it is included with the npm package.
- I have tested using the demoApp in the test and believe it could work
- Maybe don't include it by default in npm package as it it not needed